main/FatesHistoryInterfaceMod.F90

```diff
@@ -2199,6 +2247,12 @@ subroutine update_history_dyn(this,nc,nsites,sites)
                hio_crownarea_si_pft(io_si, ft) = hio_crownarea_si_pft(io_si, ft) + &
                     ccohort%c_area 
 
+               if (ccohort%canopy_layer .eq. 1) then
```

Comment:
this is useful!

Reply:
agreed.  but should we change the units on both this and the preceding variable (which becomes `PFTcrownarea`) to be m2/m2 rather than m2/ha?